lib/Runtime/Library/IntlEngineInterfaceExtensionObject.cpp

```diff
@@ -920,18 +920,10 @@ namespace Js
 
         Assert(numberFormatter);
 
-        // TODO (doilij): Render signed zero.
-        // TODO(jahorto): isDecimalPointAlwaysDisplayed seems like a Windows-only construct
-        bool isDecimalPointAlwaysDisplayed = false;
-        if (GetTypedPropertyBuiltInFrom(options, __isDecimalPointAlwaysDisplayed, JavascriptBoolean))
-        {
-            isDecimalPointAlwaysDisplayed = JavascriptBoolean::FromVar(propertyValue)->GetValue();
```

Comment:
Did we determine this is unused in WinGlob, too? Maybe you can remove it from Intl.js and the WinGlob path.

Reply:
It is never referenced from Intl.js at all --  the only references in the entire project are the GetTypedProperty call that remains in WinGlob-specific code and the entry in JnDirectFields.h. So, I don't think its ever actually set, and we can probably remove it for WinGlob too. Should that happen in this pull or a separate one?

Reply:
Your call on removing the dead code. If it's truly not used, I would lean towards removing it now.